mappings/net/minecraft/entity/mob/MobEntity.mapping

```diff
@@ -68,6 +68,7 @@ CLASS aho net/minecraft/entity/mob/MobEntity
 		ARG 0 stack
 	METHOD f setTarget (Lahn;)V
 	METHOD n setCanPickUpLoot (Z)V
+	METHOD o addGoals ()V
```

Comment:
Why not `initGoals` ?

Reply:
They both work the same, imo. Any reason for choosing one over the other?

Reply:
We already have `initAttributes` and `initDataTracker`, so in my opinion that method should be named similarly.

Reply:
Good point, will change. 👍